src/spikeinterface/widgets/traces.py

```diff
@@ -149,6 +151,11 @@ def __init__(
             )
             time_range[1] = t_end
 
+        if time_range[0] < t_start or time_range[1] < t_start:
+            raise ValueError(f"All time_range values must be greater than {t_start}")
+        if time_range[1] <= time_range[0]:
+            raise ValueError("time_range[1] must be greater than time_range[0]")
```

Comment:
```suggestion
            raise ValueError("`time_range[1]` must be greater than `time_range[0]`")
```
maybe not ticks? I guess this is an edge case where it is indexing of the variable and not the variable itself?